### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by jasonmoonen**
1. Restored the population level.
2. Players inside a shelter should no longer receive damage, unless the door is open, of course.

### CODE REVIEW

File: src/servers/ZoneServer2016/handlers/commands/commands.ts [hunk 1]
```diff
@@ -158,7 +160,7 @@ export const commands: Array<Command> = [
   },
   {
     name: "pop",
-    permissionLevel: PermissionLevels.DEFAULT,
+    permissionLevel: PermissionLevels.ADMIN,
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/handlers/commands/commands.ts:163]: Isn't it a bit much? Like i guess you will remove every logs that display world pop? I'm ok for the main menu but that's all were not gonna do stuff like that to give the illusion there is players on the server

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

File: src/servers/ZoneServer2016/handlers/commands/commands.ts (hunk 1 above)
Comment (QuentinGruber) [reply src/servers/ZoneServer2016/handlers/commands/commands.ts:163]: A better idea could be do display other data too not only the actual player count but maybe the 24h connections ?

Comment (QuentinGruber) [reply src/servers/ZoneServer2016/handlers/commands/commands.ts:163]: > A better idea could be do display other data too not only the actual player count but maybe the 24h connections ?

http://charts.h1emu.com/Last%20month%20connections.html

Comment (LegendsNeveerrDie) [reply src/servers/ZoneServer2016/handlers/commands/commands.ts:163]: Even with the changes they can view the POP with Charts and the Website. Thats all thats needed. 

We are not removing it fully. Put in mind again daybreak had No charts, no pop command, no website display and servers didn't have the pop shown until 2018 as they never had that many players.

Players have told me they hated the POP count being shown too from day one, players would use it every 20mins to an hour. Im going on player feedback and I asked JSON to push this for me. :) 

Looking back at Jan 2023's update display was good without them.

**Review by LegendsNeveerrDie — APPROVED** [review]
Im with the feedback on this one. Its good changes to help the servers POP. :)

File: src/packets/LoginUdp/LoginUdp_11/loginpackets.ts [hunk 2]
```diff
@@ -28,9 +28,10 @@ const serverField: any[] = [
   { name: "descriptionId", type: "uint32" },
   { name: "reqFeatureId", type: "uint32" },
   { name: "serverInfo", type: "string" },
+  { name: "populationLevel", type: "uint32", defaultValue: 0 },
```
[the remark this answers is not in the document]
Comment (jasonmoonen) [reply src/packets/LoginUdp/LoginUdp_11/loginpackets.ts:31]: I've already asked Eavilaswayce to update Asset_256

**Review by QuentinGruber — APPROVED** [review]

File: src/servers/ZoneServer2016/handlers/commands/commands.ts (hunk 1 above)
Comment (QuentinGruber) [reply src/servers/ZoneServer2016/handlers/commands/commands.ts:163]: ok sounds good to me

**Pull request merged**